config/routes.rb

```diff
@@ -314,7 +314,7 @@
     mount VeteranConfirmation::Engine, at: '/veteran_confirmation'
   end
 
-  mount VAOS::Engine, at: '/v0/vaos'
+  mount VAOS::Engine, at: '/'
```

Comment:
I already suspect that this is going to cause grief for code reviewers. ughh... we might have to go:

/vaos/v0 

because I'm pretty sure '/' isn't going to fly.

Reply:
I definitely think we should have the ability to namespace v1 without introducing a new module / engine, but I also see value in having the root level authority happen in the main config/routes.rb not in the modules/vaos/config/routes.rb 

@johnpaulashenfelter I'm going to defer to you on guidance here on how you want us to proceed. We can probably do /v0/vaos/fhir if we have to, but ultimately this problem will surface in the future and it would be a good time to reconcile how we do routing now to better support versioning from within the modules -- ideally without having to introduce a new module / engine.

Reply:
@kreek I think it would be helpful to introduce some comprehensive routing specs for existing routes as a separate PR. Lets get that merged until we can get questions addressed here. Those routing specs will be helpful in ensuring we don't inadvertently brake anything based on whatever approach VSP agrees.

Reply:
@saneshark I'm just thinking through this, and i wonder a bit about just going with `/vaos/whatever_your_convention_needs_to_be` so the numbering is in the module, not the API.

Want to talk to identity a little since they have a similar issue of versioning, but they're also a shared resource.

Reply:
@johnpaulashenfelter there was some discussion about this in slack and that was the direction we were leaning https://dsva.slack.com/archives/CBU0KDSB1/p1586891137363400